photon-server/src/main/java/org/photonvision/common/util/TestUtils.java

```diff
@@ -165,6 +166,10 @@ public static Path getDotBoardImagesPath() {
         return getResourcesFolderPath().resolve("calibrationBoardImages");
     }
 
+    public static File getHardwareConfigJson() {
+        return getResourcesFolderPath().resolve("hardware").resolve("HardwareConfig.json").toFile();
```

Comment:
move to root folder

Reply:
This is getting the test config json file for unit tests. Not sure how I'd move it to the root folder.